arangod/Aql/EnumerateCollectionExecutor.cpp

```diff
@@ -158,13 +169,13 @@ std::pair<ExecutionState, EnumerateCollectionStats> EnumerateCollectionExecutor:
 
   while (true) {
```

Comment:
Why is the old code still active?
Can we remove it?
I know why it is active in Calculation still, bit index does not need to have it any more.